.github/workflows/ci.yml

```diff
@@ -7,35 +7,53 @@ on:
   schedule:
     # Run every morning to detect flakiness and broken dependencies
     - cron: '17 8 * * *'
+env:
+  # if changing this variable, also change jobs.rhel.container to match
+  ROS_DISTRO: galactic
 
 jobs:
-  ci_binary:
-    name: Galactic binary job
+  ros_industrial_ci:
+    name: industrial ci
     runs-on: ubuntu-latest
     strategy:
      matrix:
-       env:
-         - {ROS_DISTRO: galactic, ROS_REPO: main}
-         - {ROS_DISTRO: galactic, ROS_REPO: testing}
+      ROS_REPO: [main, testing]
     env:
      UPSTREAM_WORKSPACE: .github/workspace.repos
     steps:
-     - uses: actions/checkout@v1
+     - uses: actions/checkout@v2
      - uses: ros-industrial/industrial_ci@master
-       env: ${{matrix.env}}
+       env:
+        ROS_REPO: ${{ matrix.ROS_REPO }}
+
+  rhel:
```

Comment:
do we always want to run this? I think it's better to use it as a flagging mechanism on master, don't think we need it on every PR

Reply:
I'm not sure I understand what you mean as a flagging mechanism. So you would prefer to get notified after a PR was merged whether it was RHEL compatible? I agree that, because RHEL is on tier 2 ROS support, that we may not want to necessarily slow down PRs but wouldn't you want to know if a PR is bringing in code incompatible with RHEL? Then you could selectively ignore that failure. I would think not having that information apparent before merge would be more confusing after the fact.

What is your ideal way of seeing this information?